### ISSUE TITLE
Renders HTTP 500 when enabled.

### ISSUE THREAD

**Issue by mwilliams** [brynary/rack-bug#1]
Enabling rack-bug with the bookmarklet returns 500's for any requests made against my app, disabling it returns to normal functionality.  Any thoughts?

Console output:
https://gist.github.com/13e071c3827d1c86004e

**Comment by brynary** [comment]
Thanks for the report.

Try master now and let me know if it's fixed. I added a #to_s which I hope will resolve this.

**Comment by mwilliams** [comment]
Pulled down master and saw your change but still receiving a 500.
https://gist.github.com/13e071c3827d1c86004e

**Comment by yaroslav** [comment]
Fixed in my fork. Rails does not have Rails::Info in production or whatever. Awaiting brynary merge

**Comment by mwilliams** [comment]
Excellent!  Thanks for the awesome work guys, it's much appreciated!

**Comment by markoa** [comment]
Please note that I had to change two lines in lib/rack/bug/panels/sql_panel/query.rb to boot Mongrel with a Rails 2.3.2 app:

```
    def filtered_backtrace
      @filtered_backtrace ||= @backtrace.map { |l| l.to_s.strip }.select do |line|
        line.starts_with?(Rails.root.to_s) &&
        !line.starts_with?(Rails.root.join("vendor").to_s)
      end
    end
```

**Comment by lukemelia** [comment]
I believe this is fully resolved with 0.3.0
